src/components/OSCALLoaderForm.js

```diff
@@ -135,6 +117,21 @@ export default function OSCALLoaderForm(props) {
             </FormControl>
           </Grid>
         )}
+        {process.env.REACT_APP_REST_BASE_URL && (
+          <Grid item xs={2}>
```

Comment:
When not in REST mode the toggle overlaps the URL input.

Reply:
Improved the layout in: https://github.com/EasyDynamics/oscal-react-library/pull/265/commits/926572c0359647b4541a790395fb3d51995f1b62